bundle/terranova/tnresources/job.go

```diff
@@ -90,7 +90,7 @@ func makeCreateJob(config jobs.JobSettings) (jobs.CreateJob, error) {
 		Trigger:              config.Trigger,
 		UsagePolicyId:        config.UsagePolicyId,
 		WebhookNotifications: config.WebhookNotifications,
-		ForceSendFields:      filterFields[jobs.CreateJob](config.ForceSendFields),
+		ForceSendFields:      filterFields[jobs.CreateJob](config.ForceSendFields, "AccessControlList"),
```

Comment:
Why did we add "AccessControlList" to the ForceSendList here? To enforce the zero value always and match TF semantics?